google/cloud/bigtable/table.cc

```diff
@@ -15,7 +15,7 @@
 #include "google/cloud/bigtable/table.h"
 #include "google/cloud/bigtable/internal/bulk_mutator.h"
 #include "google/cloud/bigtable/internal/grpc_error_delegate.h"
-#include "google/cloud/bigtable/internal/make_unique.h"
+#include "google/cloud/internal/make_unique.h"
```

Comment:
It seems that `make_unique.h` file is included but never used, this inclusion can be avoided.

Reply:
Done.